.env.example

```diff
@@ -1 +1,4 @@
-API_KEY=lbK5e0cae7xf494P3c8Q1od19h41b3fa973
+# This is an example Amberdata API Key
+# If don't have one already you can obtain one
+# by navigating to amberdata.io/pricing
+API_KEY=UAK0000000000000000Q1od19h41b3fa973
```

Comment:
@taylorjdawson is this a real key? or fake example?

Reply:
It is a fake example